web/sites/default/config/core.entity_form_display.node.guide.default.yml

```diff
@@ -4,6 +4,7 @@ status: true
 dependencies:
   config:
     - field.field.node.guide.field_body
+    - field.field.node.guide.field_body_content
```

Comment:
Since we are moving this content type into information. We don't need to add the field to the guide content type. (In fact, it might be confusing for migrators.)